modular_skyrat/modules/reagent_forging/code/forge_items.dm

```diff
@@ -69,6 +69,8 @@
 	var/average_wait = 1 SECONDS
 	///the path of the item that will be spawned upon completion
 	var/spawn_item
+	//because who doesn't want to have a plasma sword?
+	material_flags = MATERIAL_EFFECTS | MATERIAL_ADD_PREFIX | MATERIAL_GREYSCALE | MATERIAL_COLOR | MATERIAL_AFFECT_STATISTICS
```

Comment:
I kinda want to know what the effects can be, from the different materials, and not for this to be just totally up in the air.

Reply:
changes durability, maybe force? does some additional effects depending on the material-- all is known on the on_applied for each material datum.